### PR TITLE
Set ROTATED_SYSTEM_SECRET to old secret as speficied in docs.

### PR DESCRIPTION
**Pull request by prateek1192**
## Related issue

#1187 

## Checklist

- [x] I have read the [contributing guidelines](CONTRIBUTING.md)
- [x] I confirm that this pull request does not address a security vulnerability. If this pull request addresses a security
vulnerability, I confirm that I got green light (please contact [hi@ory.sh](mailto:hi@ory.sh)) from the maintainers to push the changes.
- [x] I signed the [Developer's Certificate of Origin](https://github.com/ory/keto/blob/master/CONTRIBUTING.md#developers-certificate-of-origin)
by signing my commit(s). You can amend your signature to the most recent commit by using `git commit --amend -s`. If you
amend the commit, you might need to force push using `git push --force HEAD:<branch>`. Please be very careful when using
force push.
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation within the code base (if appropriate)
- [x] I have documented my changes in the [developer guide](https://github.com/ory/docs) (if appropriate)

### CODE REVIEW

File: cmd/cli/handler_migrate.go [hunk 1]
```diff
@@ -141,7 +141,7 @@ ROTATED_SYSTEM_SECRET=%s hydra serve ...
 
 If you wish that OAuth 2.0 Access and Refresh Tokens issued with the old secret are revoked, simply omit environment variable
 ROTATED_SYSTEM_SECRET. This will NOT affect OpenID Connect ID Tokens!
-`, newSecret)
+`, oldSecret)
```
Comment (aeneasr) [inline cmd/cli/handler_migrate.go]: Line 138 `variable ROTATED_SYSTEM_SECRET to the new secret:` should also be `variable ROTATED_SYSTEM_SECRET to the **OLD** secret:`

Comment (aeneasr) [reply cmd/cli/handler_migrate.go]: ping @prateek1192 :)

Comment (prateek1192) [reply cmd/cli/handler_migrate.go]: Thanks @aeneasr :)

**Comment by aeneasr** [comment]
Thank you!

**Pull request merged**